target/linux/realtek/patches-6.12/740-dt-bindings-net-realtek-rtl82xx-document-mdi-swap-en.patch

```diff
@@ -0,0 +1,30 @@
+From 746495d8bf2a294ae33b4b4aa5f7c1ccdf7a27ab Mon Sep 17 00:00:00 2001
```

Comment:
Small but very helpful enhancement. I had some issues to downport my last RTL8226 patches. So I was thankful that all patches are managed in the generic backport/pending subfolders.

Can you move them over there too?

Reply:
If you look at pending-x.xx you see several patches that are older than 2 years. Even in the 720-realtek range. So for me pending is like "maybe some day they get upstream if someone cares".